spec/view/index_user_view_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'User features', type: :system do
+  subject do
+    @user = User.create(
+      name: 'Lilly',
+      posts_counter: 3,
+      bio: 'web developer',
+      photo: 'https://images.unsplash.com/photo-1531123414780-f74242c2b052?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8OTJ8fHBlcnNvbnxlbnwwfHwwfHw%3D&auto=format&fit=crop&w=500&q=60'
+    )
+  end
+
+  describe 'User Index page', js: true do
+    it 'displays the username of all other users.' do
+      visit users_path(subject)
+      expect(page).to have_content('Lilly')
+    end
+
+    it 'displays the the profile picture for each user.' do
+      visit users_path(subject)
+      page.has_content?('Screenshot Image')
+      expect(page).to have_css('img')
+    end
+
+    it 'display the user bio' do
+      visit users_path(subject)
+      page.has_content?(@user.bio.to_s)
+    end
+
+    it 'When I click on a user, I am redirected to that user show page.' do
+      visit users_path(@user)
+      expect(current_path).to eq(users_path(@user))
+    end
+  end
+end
```

Comment:
- [ ] When I run the tests for this project using `rspec spec/view`, only this file is passing. The rest are failing. I think this may do with the fact that the other files do not have created `users`, and/or `posts`. I think you are also using the same database for development and testing so your screenshots are passing because of records from while developing the app, but tests typically should run and pass on their own.

   Consider emptying your DB and re-cloning the project to reproduce the error.

  Only the 4 tests in this file are passing | See the error message I get
  :---:|:---:
  ![djdj](https://user-images.githubusercontent.com/36542986/208867065-29bccfe5-262e-4f5b-a0ca-74bb1f9b02f8.jpg) | ![fjfwi](https://user-images.githubusercontent.com/36542986/208867085-15827b95-089e-4364-933f-3daff0059f71.jpg)